widgetastic_manageiq/__init__.py

```diff
@@ -3893,11 +3894,15 @@ def selected_field(self):
         else:
             return self.selected_field_element.text.encode("utf-8").strip()
 
-    def add(self, subfolder):
-        self.add_subfolder()
-        wait_for(lambda: self.selected_field_element is not None, num_sec=5, delay=0.1)
-        self.browser.double_click(self.selected_field_element, wait_ajax=False)
-        self.browser.handle_alert(prompt=subfolder)
+    def add(self, folder_name):
+        self.add_button.click()
+        wait_for(lambda: self.browser.elements(self._new_field) is not None, num_sec=5, delay=0.1)
```

Comment:
is it ok to use like `bool(self.browser.elements(self._new_field))`? actually elements return `list` of web element

Reply:
I could use that bool statement, I was just trying to make it work so I didn't realize I could have also enhanced that piece of code. Will change that!